tools/worker-runner/cfg/workerconfig.go

```diff
@@ -102,6 +102,10 @@ func merge(v1, v2 interface{}) interface{} {
 	arr1, arr1ok := v1.([]interface{})
 	arr2, arr2ok := v2.([]interface{})
 	if arr1ok && arr2ok {
+		// check for overflw
```

Comment:
nit
```suggestion
		// check for overflow
```

Reply:
I feel like at this point I should claim that I leave these here for you intentionally